yotta/data/example-selftest/README.md

```diff
@@ -1,82 +0,0 @@
-# mbed TLS Selftest Example
```

Comment:
We don't have any selftest example in [the new examples repo](https://github.com/ARMmbed/mbed-os-example-tls). Do we want to migrate this example?

Reply:
we have in the [standalone](https://github.com/ARMmbed/mbedtls/blob/development/programs/test/selftest.c) and in [Mbed OS](https://github.com/ARMmbed/mbed-os/blob/master/TESTS/mbedtls/selftest/main.cpp), so I don't think there is a need for that

Reply:
OK, it moved to the `mbed-os` repo. That explains why I couldn't find it in the new examples repo.